modules/apps/object/object-admin-rest-impl/src/main/java/com/liferay/object/admin/rest/internal/resource/v1_0/ObjectDefinitionResourceImpl.java

```diff
@@ -287,7 +287,12 @@ public ObjectDefinition postObjectDefinition(
 						objectDefinition.getObjectDefinitionSettings(),
 						_objectDefinitionSettingLocalService),
 					transformToList(
-						objectDefinition.getObjectFields(),
+						ArrayUtil.filter(
+							objectDefinition.getObjectFields(),
+							objectField -> !StringUtil.equals(
+								objectField.getBusinessTypeAsString(),
+								ObjectFieldConstants.
+									BUSINESS_TYPE_AGGREGATION)),
```

Comment:
@JhossephG extract `objectFields` to a variable, apply the same logic applied to custom object definition and then apply to use in both system and custom.